config_files/survey_year_2019_config.yml

```diff
@@ -4,94 +4,81 @@
-  catch_CAN_filename: Biological/CAN/2019_biodata_catch_CAN.xlsx
-  catch_CAN_sheet: biodata_catch_CAN
-  ##############################################################################
-
-
-  ##############################################################################
-  # Paths to stratification files #
-  #################################
-
-  # file that relates the stratification to the Haul
-  # The two stratification types are found in two sheets: "Base KS" and "INPFC"
-  strata_filename: Stratification/US_CAN strata 2019_final.xlsx
-  strata_sheetname: Base KS
-
-  # file that defines the geographic definition of strata
-  # The two stratification types are found in two sheets: "stratification1" and "INPFC"
-  geo_strata_filename: Stratification/Stratification_geographic_Lat_2019_final.xlsx
-  geo_strata_sheetname: stratification1
-
-  # file that provides NASC values, which do not include age1 values
-  nasc_no_age1_filename: Exports/US_CAN_detailsa_2019_table2y+_ALL_final - updated.xlsx
-  nasc_no_age1_sheetname: Sheet1
-
-  # file that provides NASC values, which include all ages
-  nasc_all_ages_filename: Exports/US_CAN_detailsa_2019_table1y+_ALL_final - updated.xlsx
-  nasc_all_ages_sheetname: Sheet1
-  ##############################################################################
-
-
-  ##############################################################################
-  # Paths to Kriging files #
-  ##########################
-
-  # Mesh filename representing the centroids of the Kriging grid
-  mesh_filename: Kriging_files/Kriging_grid_files/krig_grid2_5nm_cut_centroids_2013.xlsx
-  mesh_sheetname: krigedgrid2_5nm_forChu
-
-  # smoothed contour used to transform the mesh points
-  smoothed_contour_filename: Kriging_files/Kriging_grid_files/transformation_isobath_coordinates.xlsx
-  #  smoothed_contour_filename: Kriging_files/Kriging_grid_files/Smoothing_EasyKrig.xlsx
-  smoothed_contour_sheetname: Smoothing_EasyKrig
-
-  vario_krig_para_filename: Kriging_files/default_vario_krig_settings_2019_US_CAN.xlsx
-  ##############################################################################
-
-
-  ##############################################################################
-  # Path to mapping between hauls and transects #
-  ###############################################
-
-  # File names for the US section
-  filename_haul_to_transect_US: Biological/US/haul_to_transect_mapping_2019.xlsx
-  haul_to_transect_US_sheetname: Sheet1
-
-  # File names for the Canada section
-  filename_haul_to_transect_CAN: Biological/CAN/haul_to_transect_mapping_2019_CAN.xlsx
-  haul_to_transect_CAN_sheetname: Sheet1
-  ##############################################################################
+##############################################################################
+# Parameters
+
+survey_year: 2019            # survey year being considered
+species_id: 22500            # target species ID for the survey year
```

Comment:
What do you think about change this to
```ini
species: hake
```
and do the mapping under the hood? Maybe with a comment that the species name will be used to find a species ID under the hood?

Reply:
I've pushed an implementation of this where the mapping is in `survey_year_2019_config.yml`; we can easily move it to the other yaml if that's our consensus. 

I used the species name "pacific_hake" to be more specific.

Reply:
I like having this `species_id_mapping`, but I think it should be in `core.py` or something similar, since these are constant and should not be changed, if we do use the number 22500 under the hood.

In fact, do we have to use a number for the species? If we just use `species: pacific_hake` and use that in the code, wouldn't that be simpler?

Reply:
We do need the code (22500 for hake) somewhere, at least at this time, because the biological data files currently include data for what I assume is all species in the hauls. `species_id`  is used to filter the data to retain only pacific hake.

I'd been thinking about this lately and I think we should the FEAT team about switching over to using biological input files that have data only for hake. That would simplify the code, too. But let's address that outside this PR.

I'll find an appropriate module for `species_id_mapping` and push a commit.

Reply:
Done.